crates/rolldown_binding/src/options/plugin/binding_plugin_context.rs

```diff
@@ -79,8 +79,8 @@ impl BindingPluginContext {
   }
 
   #[napi]
-  pub async fn emit_chunk(&self, file: BindingEmittedChunk) -> anyhow::Result<String> {
-    self.inner.emit_chunk(file.into()).await.map(|id| id.to_string())
+  pub fn emit_chunk(&self, file: BindingEmittedChunk) -> String {
+    self.inner.emit_chunk(file.into()).to_string()
```

Comment:
How about we use `block_on` on this method and keep the original one async?

Reply:
Good idea. It is a more clear way and could be reduce the overhead for rust plugins.